### PR TITLE
feat(ranger): fill usersync_install.properties

### LINKED ISSUE #369: Ranger: support important properties in usersync_install.properties
Some important properties are not templated in `usersync_install.properties`, namely `SYNC_LDAP_USER_SEARCH_FILTER` and all `SYNC_GROUP` properties.

### PR DESCRIPTION
**Pull request by Nuttymoon**
#### Which issue(s) this PR fixes:

Fixes #369 

#### Additional comments:

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: tdp_vars_defaults/ranger/ranger.yml [hunk 1]
```diff
@@ -90,13 +90,19 @@ kms_install_properties:
 usersync_install_properties:
   POLICY_MGR_URL: "https://{{ groups['ranger_admin'][0] | tosit.tdp.access_fqdn(hostvars) }}:6182"
   SYNC_SOURCE: ldap
+  SYNC_INTERVAL: 360
```
Comment (leopaul36) [inline tdp_vars_defaults/ranger/ranger.yml:93]: Should we leave this an empty string to keep the default behavior?

File: tdp_vars_defaults/ranger/ranger.yml [hunk 2]
```diff
@@ -90,13 +90,19 @@ kms_install_properties:
...
   SYNC_LDAP_BIND_DN: cn=changeme,c=fr
   SYNC_LDAP_BIND_PASSWORD: changeme
   SYNC_LDAP_SEARCH_BASE: c=fr
   SYNC_LDAP_USER_SEARCH_BASE: ou=users,c=fr
   SYNC_LDAP_USER_OBJECT_CLASS: posixAccount
   SYNC_LDAP_USER_NAME_ATTRIBUTE: uid
+  SYNC_LDAP_USER_SEARCH_FILTER: ""
+  SYNC_GROUP_SEARCH_ENABLED: "false"
+  SYNC_GROUP_USER_MAP_SYNC_ENABLED: "false"
+  SYNC_GROUP_SEARCH_BASE: ou=groups,c=fr
```
Comment (leopaul36) [inline tdp_vars_defaults/ranger/ranger.yml:104]: Should we leave this an empty string to keep the default behavior?

**Review by leopaul36 — APPROVED** [review]

File: tdp_vars_defaults/ranger/ranger.yml (hunk 1 above)
Comment (Nuttymoon) [reply tdp_vars_defaults/ranger/ranger.yml:93]: As we overwrite the `SYNC_SOURCE` to `ldap` I think we can take the default value of `SYNC_INTERVAL` for LDAP?

File: tdp_vars_defaults/ranger/ranger.yml (hunk 2 above)
Comment (Nuttymoon) [reply tdp_vars_defaults/ranger/ranger.yml:104]: I took inspiration for what is made for `SYNC_LDAP_USER_*`.
```yaml
SYNC_LDAP_USER_SEARCH_BASE: ou=users,c=fr
```
I kept the default values except for `SYNC_GROUP_SEARCH_BASE`. This has no effect as it is disabled.

**Review by rpignolet — APPROVED** [review]

**Review by gboutry — APPROVED** [review]

**Pull request merged**